Source/WebInspectorUI/UserInterface/Views/DOMTreeOutline.css

```diff
@@ -99,6 +99,7 @@ body:is(.window-inactive, .window-docked-inactive) .content-view.dom-tree.deemph
     opacity: 0.7;
 }
 
+.tree-outline.object .tree-outline.dom ol, /* Increase specificity to prevent `.tree-outline.object ol` overriding a descendant `.tree-outline.dom ol`. */
```

Comment:
this seems like something that should exist in `Source/WebInspectorUI/UserInterface/Views/ObjectTreeView.css` since that's what originally defines `.tree-outline.object`

but i can see why you'd put this here since that file doesnt have any of the `--sublist-*` variables

would it work to change the other file to be something like this instead?
```css
.tree-outline.object ol {
    display: none;
    margin: 0;
    list-style: none;
}

.tree-outline.object :not(.tree-outline.dom) ol {
    padding-inline-start: 16px;
}
```

Reply:
at the very least id suggest having this comment in both files so that if either changes we at least know that there's an odd dependency here

yet another example of why i strongly prefer child combinators `a > b` instead of descendant combinators `a b` :(